pkg/global/apply_configuration.go

```diff
@@ -70,9 +42,20 @@ func ApplyConfiguration(configuration *pb.Configuration) error {
 			view.RegisterExporter(pe)
 		}
 
-		if tracingConfiguration.AlwaysSample {
-			trace.ApplyConfig(trace.Config{DefaultSampler: trace.AlwaysSample()})
+		sampling := float64(tracingConfiguration.SampleProbability)
```

Comment:
This is only used inside the if-statement. Was your intention to also use it for the ApplyConfig() call below?

Reply:
Yes, fixed